infinispan-remote/persistence/README.md

```diff
@@ -0,0 +1,13 @@
+# Use H2 driver with in standalone mode
+
+- Download the Infinispan Server 
+- Download the h2 driver
+
+```shell
+wget https://repo1.maven.org/maven2/com/h2database/h2/2.0.202/h2-2.0.202.jar
```

Comment:
There is a new way to do this via the cli that we should promote instead.

```
Use the install command with the {brandname} Command Line Interface (CLI) to download the required drivers to the server/lib directory, for example:

install com.h2database:h2:2.0.202
```

Reply:
we can even run it with docker now. I will update the doc